common/supplemental/supplementalData.xml

```diff
@@ -5436,15 +5436,11 @@ XXX Code for transations where no currency is involved
 		<parentLocale parent="zh_Hant_HK" locales="zh_Hant_MO"/>
 	</parentLocales>
 
-	<parentLocales component="segmentations">
-		<parentLocale parent="zh" locales="zh_Hant"/>
+	<parentLocales component="segmentations grammaticalFeatures plurals">
```

Comment:
this is going to be a challenge for json because collations for example was part of the xpath, and now it's a list instead of a single item

I think I have an option that will just duplicate the data.  So it will generate a segmentations, grammaticalFeatures, plurals subkey with duplicate data (an empty set).  That will be least disruptive to consumers.

Reply:
https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-core/supplemental/parentLocales.json#L190